01-1st-glance/README.md

```diff
@@ -1,56 +1,52 @@
 
-下面是一个代码片段的例子，经典的 Hello World 应用：
+Here is an example of a snippet of the classic Hello World application:
 
-``` rust
-fn main() {
-  println!("hello, world");
+Fn  main () {
+  the println ! ( "Hello, World" ) ; 
 }
-```
+Rust affects the popular programming languages ​​including C, C ++, C #, Erlang, Haskell, OCaml, Ruby, Scheme, and Swift and the like. Rust also affected the C # 7, Elm, Idris, Swift.
 
-影响了 Rust 的流行的编程语言包括 C, C++, C#, Erlang, Haskell, OCaml, Ruby, Scheme 和 Swift 等等。Rust 也影响了 C# 7, Elm, Idris, Swift。
+Rust provides a setup program, you only need to run from the official website to download and install the program on the appropriate Caozuoxitong. Installer supports Windows, Mac and Linux (via script) 32-bit and 64-bit CPU architecture on for Apache License 2.0 or MIT Licenses.
 
-Rust 提供了安装程序，你只需要从官网下载并在相应的操作系统上运行安装程序。安装程序支持 Windows、Mac 和 Linux（通过脚本）上的32位和64位 CPU 体系架构，适用 Apache License 2.0 或者 MIT Licenses。
+Rust runs on the following Caozuojitong: Linux, OS X, Windows, FreeBSD, Android, iOS.
 
-Rust 运行在以下操作系统上：Linux, OS X, Windows, FreeBSD, Android, iOS。
+Brief mention of the history of Rust. Rust was first Mozilla employee Graydon Hoare's a personal project, starting in 2009, received support from Mozilla Institute, announced the project in 2010. Between 2010 to 2011 to achieve the bootstrap. Since then, Rust has undergone dramatic changes in the design and repeated (course extremely hard), finally released version 1.0 in May 15, 2015. In this development process, Rust built a strong vibrant community, formed a complete set of stable project contribution mechanism (which is really scary). Rust Rust now by the project developer community ( https://github.com/rust-lang/rust maintenance).
 
-简单提一下 Rust 的历史。Rust 最早是 Mozilla 雇员 Graydon Hoare 的一个个人项目，从 2009 年开始，得到了 Mozilla 研究院的支助，2010 年项目对外公布。2010 ～2011 年间实现的自举。从此以后，Rust 经历了巨大的设计变化和反复（历程极其艰辛），终于在 2015 年 5 月 15日发布了 1.0 版。在这个研发过程中，Rust 建立了一个强大活跃的社区，形成了一整套完善稳定的项目贡献机制（这是真正的可怕之处）。Rust 现在由 Rust 项目开发者社区（https://github.com/rust-lang/rust ）维护。
+Since 1.0 was released on May 15, the emergence of a large number of outstanding items (you can find on github search Rust), large companies are increasingly active participation Rust application development, as well as the open source community feedback.
 
-自 15 年 5 月 1.0 发布以来，涌现了大量优秀的项目（可以 github 上搜索 Rust 查找），大公司也逐渐积极参与 Rust 的应用开发，以及回馈开源社区。
+The book (RustPrimer) aims to provide a proper Chinese Rustaceans beginners, the latest, to understand Chinese tutorial. This book would have been perfect follow up, never stops.
 
-本书（RustPrimer）旨在为中文 Rustaceans 初学者提供一个正确、最新、易懂的中文教程。本书会一直完善跟进，永不停歇。
+This book is the result of the entire Chinese community Gongtongnuli of Rust. Wherein, Rustacean involved in the book writing and revision are (in alphabetical order):
 
-本书是整个 Rust 中文社区共同努力的结果。其中，参与本书书写及校订的 Rustacean 有（排名不分先后）：
+daogangtang (Mike Cat)
```

Comment:
I dont think name is something we could, like, adjust later, it should be something always done right.
But beyond that I whole heartedly agree with using google translate.